nuts/client/auth.go

```diff
@@ -66,6 +69,32 @@ func (c HTTPClient) GetIrmaSessionResult(sessionToken string) ([]byte, error) {
 	return testAndReadResponse(http.StatusOK, resp)
 }
 
+func (c HTTPClient) CreateDummySession(customer domain.Customer) ([]byte, error) {
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+	defer cancel()
+
+	body := nutsAuthClient.CreateSignSessionJSONRequestBody{
```

Comment:
This todo: `// todo: the contract template should be used to select the dummy attributes to add` can be found in dummy.go in the nuts-node.
If that todo is solved, this will brake since it's not using a contract?

Reply:
Probably. Good catch. Fixed it by using the same logic as the IRMA flow.